schemas/json/layout/layout.schema.v1.json

```diff
@@ -194,6 +194,12 @@
     "gridSettings": {
         "allOf": [{ "$ref": "#/definitions/gridProps"}],
         "properties": {
+              "labelGrid": {
```

Comment:
Too indented? :)

Reply:
The whole file is a mess, switching between 2 and 4 spaces indentation. Not an excuse for adding more inconcistency (6 spaces), but annoying when an autoformatter makes the file way less readable by expanding lots of arrays I'd rather keep on a single line.

Reply:
Heh, this PR just grew a little bit larger... :)
But cleanup up is good, and this cleanup looks good to me. But could you have a quick look aswell @lorang92 before I merge?

Reply:
@altinnadmin everything looks good to me! Great job @ivarne